helm/dagster/schema/schema/charts/dagster/subschema/dagit.py

```diff
@@ -16,19 +16,19 @@ class Workspace(BaseModel):
 
 
 class Dagit(BaseModel):
-    replicaCount: int
     image: kubernetes.Image
     service: kubernetes.Service
     workspace: Workspace
-    env: Dict[str, str]
-    envConfigMaps: List[kubernetes.ConfigMapEnvSource]
-    envSecrets: List[kubernetes.SecretEnvSource]
-    nodeSelector: kubernetes.NodeSelector
-    affinity: kubernetes.Affinity
-    tolerations: kubernetes.Tolerations
-    podSecurityContext: kubernetes.PodSecurityContext
-    securityContext: kubernetes.SecurityContext
-    resources: kubernetes.Resources
+    replicaCount: Optional[int] = 1
+    env: Optional[Dict[str, str]] = {}
```

Comment:
To make it easier to instantiate a basic version of this class, I followed some patterns from `user_deployments.py` and made some values optional with defaults from the `values.yaml` file.  Not sure if all these should be optional, but the defaults made testing a lot easier.